docs/src/test-reporter-api/class-reporter.md

```diff
@@ -131,6 +132,23 @@ Called on some global error, for example unhandled exception in the worker proce
 
 The error.
 
+## optional async method: Reporter.onExit
+* since: v1.33
+
+Called immediately before test runner exists. At this point all the reporters
+have recived the [`method: Reporter.onEnd`] signal, so all the reports should
+be build. You can run the code that uploads the reports in this hook.
+
+### param: Reporter.onExit.result
```

Comment:
Leftover from `onEnd`? I don't see this being passed, and we do not really need it.